### ISSUE TITLE
Adjust brightness according to the outside brightness

### ISSUE THREAD

**Issue by petetnt** [jamox/hyperambient#1]
Maybe more of a general question about the ambient sensor and Hyperterm theming: would it be possible to adjust the brightness of the theme instead of completely switching between themes when the threshold is passed?

**Comment by jamo** [comment]
Definitely, that would be awesome. Thou I'm not sure how to implement it; I don't feel like having a black opaque overlay div ontop of the view...

**Comment by jamo** [comment]
I took a quick shot on this, and just blindly darkening/brightening with https://github.com/Qix-/color#manipulation didn't really yield the wanted result with the solarized dark/light themes.

**Comment by albinekb** [comment]
I think this needs custom themes, i can try to hack something together

**Comment by jamo** [comment]
Awesome. I wish we find a way that would fit to all themes w/o any modifications to the themes...

Btw, I just added support for customizing light and dark themes: https://github.com/jamox/hyperambient#customizing - so now there is a config file where this feature could easily be configured.

**Comment by albinekb** [comment]
Cool! Yes it would be awesome to make it work with all themes. Maybe with just a bit of logic we can get it to work. We probably need to check all the colors of the theme, use something to determine contrast between them to be able to change the color correctly
